src/main/java/com/spotify/fmt/Serialization.java

```diff
@@ -0,0 +1,77 @@
+/*-
+ * -\-\-
+ * Flo Workflow Definition
+ * --
+ * Copyright (C) 2016 - 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
How does it work combining licenses like this?

Reply:
This is all our source code and we can relicense it. Let’s remove or align these license headers.

Reply:
Have no idea what is most appropriate :)